tensorflow_quantum/python/optimizers/utils.py

```diff
@@ -0,0 +1,103 @@
+# Copyright 2020 The TensorFlow Quantum Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Utils for tensorflow quantum optimizers
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from operator import mul
+from functools import reduce
+import tensorflow as tf
+
+
+def function_factory(model, loss, train_x, train_y):
+    """A factory to create a function required by tfq.optimizer.rotosolve.
+    This function is originally defined for l-bgfs minimizer for tensorflow

```

Comment:
one blank line please.
```
"""A factory to create a function required by tfq.optimizer.rotosolve.

This function is originally defined for l-bgfs minimizer for tensorflow
```

Reply:
Fixed in 7f8bfb40be4c6cd24439320b942af83e1a0490e0